package.json

```diff
@@ -7,15 +7,16 @@
     "type": "git",
     "url": "https://github.com/dai-shi/use-context-selector.git"
   },
+  "browser": "./dist/index.umd.js",
   "main": "./dist/index.js",
-  "module": "./src/index.js",
+  "module": "./dist/index/module.js",
```

Comment:
I would add `"source": "./src/index.js",` as well as `"module": "./dist/index/module.js",`.

Reply:
There aren't a lot of real world usages for the source variable. There will be an evolution in microbundle where it will support syntax.esmodules, this basically publishes a supplementary package that is written in ES6 instead of ES5